File: Find the divisors.dart
Object divisors(int number) {
  List<int> count = [];
  for(int i=2;i<=(number/2).floor();i++){
    if(number%i==0){
      count.add(i);
    }
  }
  return count.length ==0 ? '$number is prime' : count;
}

File: Find the divisors.ts
export function divisors(integer: number): number[] | string {
  const result = Array.from({length:integer},(_,i) => i+1).filter(v => v!==1 && v!==integer && integer%v===0);
  return result.length!==0 ? result : `${integer} is prime`;
}

